api/v3/Setting.php

```diff
@@ -160,6 +160,12 @@ function civicrm_api3_setting_getoptions($params) {
     $values = Civi\Core\Resolver::singleton()->call($pseudoconstant['callback'], array());
     return civicrm_api3_create_success($values, $params, 'Setting', 'getoptions');
   }
+  elseif (!empty($pseudoconstant['optionGroupName'])) {
+    return civicrm_api3_create_success(
+      CRM_Core_OptionGroup::values($pseudoconstant['optionGroupName'], FALSE, FALSE, TRUE),
+      $params, 'Setting', 'getoptions'
+    );
+  }
```

Comment:
@eileenmcnaughton reason to add this code is we cannot call ```CRM_Core_Pseudoconstant::get(..)``` OR ```buildOptions``` here as it needs DAO/BAO name to be passed either as parameter or class respectively as stated above at line 156.